generators/server/templates/src/test/java/package/web/rest/_LogsResourceIntTest.java

```diff
@@ -19,10 +19,11 @@
 package <%= packageName %>.web.rest;
 
 <%_ if (databaseType === 'cassandra') { _%>
-import <%= packageName %>.AbstractCassandraTest;
+import <%=packageName%>.AbstractCassandraTest;
 <%_ } _%>
-import <%= packageName %>.<%= mainClass %>;
-import <%= packageName %>.web.rest.vm.LoggerVM;
+import <%=packageName%>.<%= mainClass %>;<% if (authenticationType == 'uaa') { %>
```

Comment:
Yes. But I also like the "end-of-line" type for import filter and it's used in a lot of templates.
We don't have an established rule for this. Maybe we should...